### ISSUE TITLE
Do you train the net on your own ?

### ISSUE THREAD

**Issue by ghost** [LeslieZhoa/tensorflow-YOLO1#2]

**Comment by LeslieZhoa** [comment]
I trained it with the pretrain model by myself.

**Comment by ghost** [comment]
I have a question that did you use your model to train on a custom data ? I see it is very difficult to make loss decrease

**Comment by LeslieZhoa** [comment]
No,I use the VOC data. The base model is VGG. If your dataset is not big enough, the model will converge hard. I think u could try to train your own model without the pretrain model, if possible. I hope it can help u!
